.circleci/config.yml

```diff
@@ -1104,7 +1104,7 @@ jobs:
     environment:
       <<: *base_ubuntu2204_large_env
       CMAKE_BUILD_TYPE: Debug
-      CMAKE_OPTIONS: -DCMAKE_CXX_STANDARD=20 -DUSE_CVC4=OFF
```

Comment:
Wait, doesn't this mean that `cvc5` will be used if found? Should this be the default behavior now in these images?

Reply:
`cvc5` will be used only in tests that explicitly request it.
It is now runtime dependency, not a compile-time dependency, as `CVC4` was.

Not sure what you mean by default behaviour. As I understand it, default behaviour does not change.

Reply:
Oh, right. Previously, we explicitly disabled `cvc4` on those images, so I was concerned that this change might cause issues. However, everything seems to be fine indeed.